pkgs/applications/misc/hugo/default.nix

```
@@ -34,6 +42,11 @@ buildGoModule rec {
       --zsh  <(${emulator} $out/bin/hugo completion zsh)
   '';
 
+  passthru.tests.version = testers.testVersion {
+    package = hugo;
+    command = "hugo version";
```

```suggestion
    command = "hugo version";
    version = "v{version}";
```

From https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion

> `# The output needs to contain the 'version' string without any prefix or suffix.`

---

Huh, I could've sworn I tested that :confused: